tests/model_hub_tests/pytorch/models/nightly

```diff
@@ -0,0 +1,34 @@
+hf-tiny-model-private/tiny-random-BartForCausalLM
+hf-tiny-model-private/tiny-random-BigBirdForCausalLM
+hf-tiny-model-private/tiny-random-BigBirdPegasusForCausalLM
+hf-tiny-model-private/tiny-random-BioGptForCausalLM
+hf-tiny-model-private/tiny-random-BlenderbotSmallForCausalLM
+hf-tiny-model-private/tiny-random-BlenderbotForCausalLM
+hf-tiny-model-private/tiny-random-ErnieForCausalLM
+hf-tiny-model-private/tiny-random-GPTNeoXJapaneseForCausalLM
+hf-tiny-model-private/tiny-random-MBartForCausalLM
+hf-tiny-model-private/tiny-random-MvpForCausalLM
+hf-tiny-model-private/tiny-random-PegasusForCausalLM
+hf-tiny-model-private/tiny-random-PLBartForCausalLM
+hf-tiny-model-private/tiny-random-XGLMForCausalLM
+PrunaAI/hf-tiny-model-private-tiny-random-BloomForCausalLM-bnb-4bit-smashed
+PrunaAI/hf-tiny-model-private-tiny-random-BloomForCausalLM-bnb-8bit-smashed
+PrunaAI/hf-tiny-model-private-tiny-random-BloomForCausalLM-HQQ-2bit-smashed
+hf-internal-testing/tiny-random-PersimmonForCausalLM
+hf-internal-testing/tiny-random-BartForCausalLM
+hf-internal-testing/tiny-random-GPTBigCodeForCausalLM
+hf-internal-testing/tiny-random-XGLMForCausalLM
+hf-internal-testing/tiny-random-PegasusForCausalLM
+hf-internal-testing/tiny-random-MBartForCausalLM
+hf-internal-testing/tiny-random-BigBirdPegasusForCausalLM
+hf-internal-testing/tiny-random-BigBirdForCausalLM
+hf-internal-testing/tiny-random-MegaForCausalLM
+hf-internal-testing/tiny-random-RobertaPreLayerNormForCausalLM
+hf-internal-testing/tiny-random-BioGptForCausalLM
+hf-internal-testing/tiny-random-ProphetNetForCausalLM
+hf-internal-testing/tiny-random-PLBartForCausalLM
+hf-internal-testing/tiny-random-MegatronBertForCausalLM
+hf-internal-testing/tiny-random-GPTNeoXJapaneseForCausalLM
+hf-internal-testing/tiny-random-ErnieForCausalLM
+hf-internal-testing/tiny-random-BlenderbotForCausalLM
+hf-internal-testing/tiny-random-BlenderbotSmallForCausalLM
```

Comment:
do we have extra value from `nightly` list? As I remember, we dropped `smashed` models in GenAI cont. batching.

Reply:
@ilya-lavrenov , can please elaborate on your question. Don't really understand it.